modules/ROOT/pages/reproducible-builds.adoc

```diff
@@ -0,0 +1,11 @@
+= Reproducible Builds
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You should be able to build a particular version of your Mule project at any point of time. This facilitates maintenance as the versions of your projects in production will certainly not be the latest ones.
+
+The following will help you achieve this goal:
+
+* Source control all your Mule project, like any other project (remember not to check-in Eclipse specific files). Branching, merging and tagging are all applicable practices to Mule projects
```

Comment:
Is "Source control" a verb? Recommend revision to make "source control" a noun.

"check-in" is not hyphenated when used as a verb: https://www.merriam-webster.com/dictionary/check%20in

"are all applicable practices to Mule projects" ->  "apply to"  or "all apply to"?